dockerfiles/typescript.Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Build in a full featured container
 ARG TARGETARCH
-FROM --platform=linux/$TARGETARCH node:20-bullseye AS build
+FROM --platform=linux/$TARGETARCH node:20-bookworm AS build
```

Comment:
Newest Debian release. I [was seeing some errors](https://github.com/temporalio/omes/actions/runs/16783575389/job/47528419882?pr=180) and this seemed to have fixed it.